src/map/entities/baseentity.h

```diff
@@ -284,6 +284,8 @@ class CBaseEntity
     uint32 GetLocalVar(std::string var);
     void   SetLocalVar(std::string var, uint32 val);
 
+    std::map<std::string, uint32> GetAllLocalVars();
```

Comment:
The naming here isn't particularly helpful, `GetLocalVars` is fine, unless there are other functions that return a filtered version of the map

Reply:
Also, the return type is quite long, you can use `auto GetLocalVars() -> std::map<std::string, uint32>;`

Reply:
> The naming here isn't particularly helpful, `GetLocalVars` is fine, unless there are other functions that return a filtered version of the map

Thanks, I updated the PR, hopefully this is ok now.